dev/ci/platform-windows.bat

```diff
@@ -0,0 +1,110 @@
+REM @ECHO OFF
+
+REM SET ARCH=64
+REM SET PLATFORM="https://github.com/coq/platform/archive/v8.13.zip"
+REM SET CI_PROJECT_DIR="C:\root"
+
+REM This script builds a minimal Windows platform on Gitlab
+
+ECHO "Start Time"
+TIME /T
+
+REM List currently used cygwin and target folders for debugging / maintenance purposes
+
+ECHO "Currently used cygwin folders"
+DIR C:\ci\cygwin*
+ECHO "Currently used target folders"
+DIR C:\ci\coq*
+ECHO "Root folders"
+DIR C:\
+
+SET CYGROOT=C:\ci\cygwin%ARCH%
+SET DESTCOQ=C:\ci\coq%ARCH%
+SET CYGCACHE=C:\ci\cache\cgwin
+
+CALL :MakeUniqueFolder %CYGROOT% CYGROOT
+CALL :MakeUniqueFolder %DESTCOQ% DESTCOQ

```

Comment:
This is no longer used for the platform - the dest folder is ~/.opam

Reply:
Hum, my understanding was that this was needed in order to not get clash between jobs.